worlds/messenger/portals.py

```diff
@@ -204,30 +187,51 @@
 }
 
 
+REGION_ORDER = [
```

Comment:
any reason why this moved?

Reply:
logical organization. hard to see in the diff but this list was in the middle of the possible portal outputs. probably should've been a separate PR since it doesn't have an affect on the issues here, but the main purpose of this was fixing plando and users can use the other collections as references for their plando